feature-group-definitions/grid.yml

```diff
@@ -1,13 +1,24 @@
 spec: https://drafts.csswg.org/css-grid-3/
 caniuse: css-grid
+status:
+  is_baseline: true
+  since: "2020-06-02" # since Firefox 77
```

Comment:
This might look a little funny.

Firefox mostly implements `grid` from version 52. Alternative names and a few exceptional features delays full, unambiguous support to 76. The main problem is `repeat` values for `grid-template-{rows,columns}`: there are multiple notes and partial implementations.

Were it not for `repeat` then Edge 80 would mark the baseline cutoff, four months earlier.